orchagent/aclorch.cpp

```diff
@@ -2913,6 +2920,12 @@ sai_object_id_t AclOrch::getTableById(string table_id)
 {
     SWSS_LOG_ENTER();
 
+    if (table_id.empty())
+    {
+        SWSS_LOG_WARN("table_id is empty");
+        return SAI_NULL_OBJECT_ID;
+    }
+
```

Comment:
do you know why it will pass the `table_id == m_mirrorTableId || table_id == m_mirrorV6TableId`? Have you also inserted an ACL table first with an empty name string?

Reply:
ACL table was fine, but the ACL RULE has no Table specified as mentioned
"ACL_RULE": {
    "|RULE1": {
        "IP_TYPE": "IPv4ANY",
        "PACKET_ACTION": "DROP",
        "PRIORITY": "0"
     }
}

We don't have Mirror table configuration, thus hit "table_id == m_mirrorTableId" and  "table_id == m_mirrorV6TableId"